test/core/iomgr/poller/eventmanager_libuv_test.cc

```diff
@@ -0,0 +1,58 @@
+
+/*
+ *
+ * Copyright 2019 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include "src/core/lib/iomgr/poller/eventmanager_libuv.h"
+
+#include <grpc/grpc.h>
+#include <grpc/support/time.h>
+
+#include "test/core/util/test_config.h"
+
+using grpc::experimental::LibuvEventManager;
+
+static void test_manager_allocation() {
```

Comment:
Even though this is core, we do now allow gtest for core. I'd encourage you to do so so that the tests look more "normal".

Reply:
Done.